trezor/trezor-suite
# Feat/udev install

fix https://github.com/trezor/trezor-suite/issues/3664

@trezor/qa scenarios

1. remove udev rules from linux machine:
`pkexec rm /etc/udev/rules.d/51-trezor.rules` or `sudo rm /etc/udev/rules.d/51-trezor.rules`

2. run suite app, connect Trezor

3. click "install rules automatically` 

3. a) if you cancel Troubleshooting tips should unfold automatically offering you alternative: packages to download

3. b) if you succeed Troubleshooting tips should change to "reconnect device"

4. check if offerred packages are corresponding with your linux distribution (rpm/deb)

suite-web is not able to do automatic install therefore it only offers packages to download

![Screenshot from 2021-08-25 16-36-07](https://user-images.githubusercontent.com/3435913/130813080-9c854e70-16ff-4394-9f95-165ba21698a1.png)
![Screenshot from 2021-08-25 16-36-25](https://user-images.githubusercontent.com/3435913/130813103-0e4b6eff-c085-42bb-b718-b74e1d1e434d.png)
![Screenshot from 2021-08-25 16-36-56](https://user-images.githubusercontent.com/3435913/130813114-e8d89ef4-b5ea-496f-a8ac-a876a61e747d.png)
